safety/util.py

```diff
@@ -54,7 +54,14 @@ def read_requirements(fh, resolve=False):
             # if this is a tempfile, skip
             if is_temp_file:
                 continue
-            filename = line.strip("-r ").strip("--requirement").strip()
```

Comment:
The bug was that `strip([chars])` treats `chars` as a list of chars to each separately remove, rather than a single strip to remove.

`"-r requirements_2.txt".strip("-r ")` returns `equirements.txt`